enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -153,34 +154,29 @@ OLM is a subscriber to these `Proxy` settings -- it forwards the settings to the
 container will automatically get the required `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables on startup.
 In fact, OLM will update and restart the operator pod with proper environment variables if the `Proxy` resource changes.
 
-In proxy-enabled clusters, WMCO will read the values of the 3 proxy variables from its own environment and store them in the 
-[`windows-services` ConfigMap](./health-management.md#services-configmap).
-In clusters without a global proxy, these variables will not be present in the services ConfigMap.
-This will be done by adding a `EnvironmentVars` key to the top level of the services ConfigMap spec.
-The new key will be an array of name-value pairs representing environment variables to be set on Windows instances.
-```json
-{
-  "EnvironmentVars": [
-    {
-      "name": "name of the environment variable",
-      "value": "value the variable should be set to on the instance"
-    }
-  ],
-  "Services": "<existing spec>",
-  "Files": "<existing spec>",
-}
-```
+In proxy-enabled clusters, WMCO will read the values of the 3 proxy variables from its own environment and store them in 
+the name-value pairs within the `EnvironmentVars` key found in the [`windows-services` ConfigMap](./health-management.md#services-configmap) spec.
+However, in clusters without a global proxy, these variables will not be present in the services ConfigMap.
+WMCO also specifies a list of environment variables monitored by WICD through the `WatchedEnvironmentVars` key in the
+services ConfigMap spec.
+
+WICD, which uses the ConfigMap data to configure Windows services, now has the added responsibility of ensuring
+the `EnvironmentVars` are correctly set on the node and its managed services. WICD checks if any env vars have changed 
+by comparing each variable's value on the node to the expected value in the ConfigMap.
+If there is a discrepancy, WICD will update the env vars through a [Windows syscall](https://pkg.go.dev/golang.org/x/sys/windows#SetEnvironmentVariable)
+which sets the environment variables system-wide, also known as the `Machine` level in Windows. 
+
+There are 3 levels to Windows environment variables: `Machine`, `User`, and `Process`. `Process` inherits from 
+`User`, which inherits from the top level of `Machine`, and sub-processes inherit from their parent process. 
+In order to route all external node traffic through the proxy, we must set the proxy variables at the `Machine` level.
+This will also allow the required services (including kubelet and containerd) to inherit the desired values.
+
+Additionally, WICD ensures if the environment variable names in the `EnvironmentVars` key are present in the
+`WatchedEnvironmentVars` list found in the ConfigMap. If there is a mismatch, WICD will remove the unmanaged
+environment variable from the Windows OS registry. 
```

Comment:
This detail needs to be in the WICD enhancement. In the proxy enhancement you just need to mention how you leverage this feature.